repo/packages/K/kafka/0/config.json

```diff
@@ -1,6 +1,35 @@
 {
-  "type": "object",
-  "properties": {},
-  "additionalProperties": false,
-  "required": []
+    "type": "object",
+    "properties": {
```

Comment:
So, do you suggest to add "kafka/" namespace, so config properties would be called, for instance, 
kafka/master, kafka/zk, etc? If you think, that this approach is pragmatic, I will add those namespace.